### PR TITLE
pyfaf: create_problems: Batch invalid reports

### PR DESCRIPTION
**Pull request by ernestask**
Seems that the create_problems action that’s been running on retrace01
for a couple of days now is spending a lot of time querying for reports.

Signed-off-by: Ernestas Kulik <ekulik@redhat.com>

### CODE REVIEW

**Comment by ernestask** [comment]
Just something I had lying around for a while.

File: src/pyfaf/actions/create_problems.py [hunk 1]
```diff
@@ -454,11 +454,9 @@ def _create_problems(self, db, problemplugin, #pylint: disable=too-many-statemen
 
             self.log_debug("Removing {0} invalid reports from problems"
                            .format(len(invalid_report_ids_to_clean)))
-            for report_id in invalid_report_ids_to_clean:
-                db_report = get_report_by_id(db, report_id)
-                if db_report is not None:
-                    db_report.problem_id = None
-                    db.session.add(db_report)
+            for db_report in get_reports_for_ids(db, invalid_report_ids_to_clean, 1000)
```
Comment (mkutlak) [inline src/pyfaf/actions/create_problems.py]:
```suggestion
            for db_report in get_reports_for_ids(db, invalid_report_ids_to_clean, 1000):
```

Comment (ernestask) [reply src/pyfaf/actions/create_problems.py]: Damn it to hell.

**Comment by ernestask** [comment]
@trapas, go to hell and retest this please

**Comment by ernestask** [comment]
test this please

File: src/pyfaf/actions/create_problems.py [hunk 2]
```diff
@@ -454,11 +454,9 @@ def _create_problems(self, db, problemplugin, #pylint: disable=too-many-statemen
 
             self.log_debug("Removing {0} invalid reports from problems"
                            .format(len(invalid_report_ids_to_clean)))
-            for report_id in invalid_report_ids_to_clean:
-                db_report = get_report_by_id(db, report_id)
-                if db_report is not None:
-                    db_report.problem_id = None
-                    db.session.add(db_report)
+            for db_report in get_reports_for_ids(db, invalid_report_ids_to_clean, 1000):
+                db_report.problem_id = None
```
Comment (xsuchy) [inline src/pyfaf/actions/create_problems.py]: Can this be updated by something like `UPDATE report SET problem_id WHERE id in <IDS>`?

Comment (ernestask) [reply src/pyfaf/actions/create_problems.py]: Hm, good point. I’ll look into that.

**Pull request merged**